docs/wallets.md

```diff
@@ -0,0 +1,41 @@
+---
+id: wallets
+title: Polkadot Wallets
+sidebar_label: Polkadot Wallets
+---
+
+## Polkadot Wallets
+
+The integration of a [wallet](https://wiki.polkadot.network/docs/en/glossary#wallet) and Polkadot
+allows for simple and easy access to private keys and signing transactions. Below are wallets
+supported by Polkadot.
+
+### Live Wallets
+
+- [Polkawallet](https://polkawallet.io/) - Mobile wallet available on IOS and Android
```

Comment:
Let's put these all in a table.

We don't want to endorse any specific wallet, but should include as much detail for people to make up their mind as possible.

You should also double-check that these work before posting them.